src/Form/FormMapper.php

```diff
@@ -93,7 +93,7 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
             $fieldDescriptionOptions['type'] = $type;
         }
 
-        if ($group['translation_domain'] && !isset($fieldDescriptionOptions['translation_domain'])) {
+        if (!isset($fieldDescriptionOptions['translation_domain']) && isset($group['translation_domain'])) {
```

Comment:
Why this change? Are you also fixing a bug?

Reply:
No, it was a change I made while covering `with()` and `tab()` methods. It's reverted now, thank you!